automatic/7zip.install/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,40 @@
+﻿Function Get-ExplorerProcessCount
+{
+  $process = Get-Process explorer -ErrorAction SilentlyContinue
```

Comment:
`-ea 0` instead of `-ErrorAction SilentlyContinue` to be consistent with other packages